penndata/views.py

```diff
@@ -149,13 +156,21 @@ def __init__(self, myType, myInfo=None, myWeight=0):
         def getCell(self):
             return {"type": self.type, "info": self.info}
 
+    def get_gsr_invites(self, username):
```

Comment:
You should just pass the user to this method instead of the username which requires a check to get the underlying user